app/v1/dist/index.html

```diff
@@ -159,9 +163,11 @@
         let stats;
         let mesh;
         let gui;
+        let downloadButton;
 
-        watchFile(path, ({ data }) => {
+        watchFile(path, (file) => {
```

Comment:
Suggest either:

watchFile(path, ({ data, translator }) =>

or:

const { data, translator } = file.

Since file itself doesn't seem to be used, I'd recommend the first option.